src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,43 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
+    
+    if len(arrA) == 0:
+        return arrB
+    if len(arrB) == 0:
+        return arrA
+    
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
-    # TO-DO
+
+    x, y = 0, 0
+    for i in range(elements):
```

Comment:
Great job using index counters for keeping track of your place in each array.  Other options like popping items and mutating the size of those arrays are definitely less optimal than how you have done it here.